src_kolko_krzyżyk/TicTacToe_expand.py

```diff
@@ -185,7 +185,35 @@ def main(self):
             else:
                 player = 0
 
-game = TicTacToe(5)
+def menu():
+    print('''\n\n\t"Simple game for two person in Tic Tac Toe.
+     Player decides how big the board is. Valid board (2 to n)
+     where board are matrix n x n. Good Luck and Have Fun."\n\n''')
+    # default matrix_choice
```

Comment:
Remove this comment and the line below. They don't do anything.

Reply:
thanks for hints. In next week i will try implement your solutions.
BTW maybe you don't know I wrote few new method earlier but i don't see them here in changes